src/app/extensions.cpp

```diff
@@ -1003,6 +1006,11 @@ ExtensionInfo Extensions::getCompressedExtensionInfo(const std::string& zipFn)
     std::string err;
     auto json = json11::Json::parse(out.str(), err);
     if (err.empty()) {
+      if (json["contributes"].is_object()) {
+        auto themes = json["contributes"]["themes"];
+        if (themes.is_array() && themes[0].is_object())
+          info.themeId = themes[0]["id"].string_value();
```

Comment:
Check all themes, one extension can have multiple items in the "themes" array (and the second one could be called `"default"`):
```suggestion
        if (themes.is_array()) {
          for (...) {
            if (themes[i]["id"].string_value() == kAsepriteDefaultThemeName) {
              info.defaultTheme = true;
              break;
            }
          }
        }
```